cfgov/jinja2/v1/_includes/organisms/content-banner.html

```diff
@@ -0,0 +1,33 @@
+{# ==========================================================================
+
+   Banner
+
+   ==========================================================================
+
+   Description:
+
+   Create an banner when given:
+
+    value:         				An object used to customize the markup.
```

Comment:
Should be fixed in https://github.com/cfpb/cfgov-refresh/pull/2932/commits/df690923fca3e24cdfd7bc3deb33ced2fb51892a